website/src/components/canvas/Sidebar.jsx

```diff
@@ -104,16 +138,34 @@ class Sidebar extends React.Component {
       <div
```

Comment:
I mentioned this in a previous CR but two wrapping divs is redundant. Just use one